neurots/extract_input/input_parameters.py

```diff
@@ -103,6 +124,14 @@ def merged_params(data):
         if method == "tmd":
             input_parameters["apical_dendrite"]["growth_method"] = "tmd_apical"
 
+    if trunk_method == "3d_angles":
```

Comment:
is it maybe possible to avoid having twice the same "if" maybe they can be put together?

Reply:
it is!